packages/backend/src/models/CatalogModel/utils/parser.ts

```diff
@@ -27,8 +29,8 @@ const parseFieldFromMetricOrDimension = (
     },
 ): CatalogField => ({
     name: field.name,
-    label: field.label,
-    description: field.description,
+    label: catalogArgs.label ?? '',
+    description: catalogArgs.description ?? '',
```

Comment:
Need your thoughts here;
the `parser` is run when `CatalogModel.search`;
Currently, this receives the explore (from db) with no changeset applied. 

With this change, we are returning the `catalog_search` column values. 
I wonder if this should be further improved to take into account the update explore so we can extend further. For example, if the explore had the changeset applied, this file would remain unchanged

lmk what you think.